hack/check_tuningPolicy.sh

```diff
@@ -45,4 +45,25 @@ for jpath in "${RATE_LIMITS[@]}"; do
      exit 1
   fi
   sleep 2
-done
\ No newline at end of file
+done
+
+./hack/retry.sh 10 3 "(${KUBECTL_BINARY} patch -n \"${INSTALLED_NAMESPACE}\" hco kubevirt-hyperconverged --type=json -p='[{"op": "remove", "path": "/spec/tuningPolicy", "value": "annotation"}]')"
+./hack/retry.sh 10 3 "(${KUBECTL_BINARY} annotate -n \"${INSTALLED_NAMESPACE}\" hco kubevirt-hyperconverged hco.kubevirt.io/tuningPolicy-)"
```

Comment:
My only concern is that I think that it should be done as an atomic operation, but if it's working, then it's OK.

/lgtm